ROLES.md

```diff
@@ -164,6 +165,45 @@ a 180 day period, than that individual may lose membership. On-going contributio
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
+## Content Reviewer
```

Comment:
This role has a generic (repo independent) name, but it is described as an istio.io specific role. If such a role makes sense in other repos then I think it needs to be described more generally. If not, then shouldn't the name be something like "Doc Reviewer", instead of "Content Reviewer", and also should it be documented here, along with all the other roles that apply to any repo?

Reply:
Sure, I don't feel strongly about "Content Reviewer", I'll make the change.